src/dotnet-uninstall/Shared/BundleInfo/BundleVersion.cs

```diff
@@ -44,7 +44,7 @@ public static BundleVersion FromInput<TBundleVersion>(string input)
                 return RuntimeVersion.FromInput(input);
             }
 
-            throw new ArgumentOutOfRangeException();
+            throw new ArgumentException();
```

Comment:
nit: even only the developer should see this exception, it is usually good to add some explaining why. So people can debug faster. You can add something like "cannot find enum matching"